elements/lisk-transactions/test/10_delegate_transaction.ts

```diff
@@ -45,7 +56,7 @@ describe('Delegate registration transaction class', () => {
 		});
 
 		it('should set the delegate asset', async () => {
-			expect(validTestTransaction.asset.username).to.eql('0x0');
+			expect(validTestTransaction.asset.username).to.eql('new_delegate');
```

Comment:
expect(validTestTransaction.asset.username).to.eql(protocolSpecDelegateFixture.testCases.input.account.transaction.asset.username); ?
more accurate expectation?

Reply:
Fixed.